pkg/api/vminfo.go

```diff
@@ -26,6 +26,10 @@ const (
 	AnnotationAutoscalingBounds   = "autoscaling.neon.tech/bounds"
 	AnnotationAutoscalingConfig   = "autoscaling.neon.tech/config"
 	AnnotationBillingEndpointID   = "autoscaling.neon.tech/billing-endpoint-id"
+
+	// ref cloud#15939; to be removed after rollout is complete.
```

Comment:
```suggestion
	// ref neondatabase/cloud#15939; to be removed after rollout is complete.
```

Question: why labels are needed?

Reply:
Labels are there mostly because I'm expecting that there will _probably_ be some issues, being able to enable only for a single VM should make it easier to test, and labels being opt-in means the release process doesn't need to be blocked on testing scaling event reporting.

It's also in part because I'd like to get #1108 released without needing to coordinate testing of this PR :sweat_smile:

This is definitely not the only way to do it, though -- I'm curious for your thoughts regardless.

Reply:
> But the global configuration is also opt-in, so we can always roll-out with scaling reporting disabled.

That's a fair point — removed the per-VM configuration in e15b277359bcdc23945db258ee87cea1482769a8